### PR TITLE
Add ERC: NFT Dynamic Traits

### PR DESCRIPTION
**Pull request by ryanio**
Discussion thread: https://ethereum-magicians.org/t/erc-7496-nft-dynamic-traits/15484

Prior PR: https://github.com/ethereum/EIPs/pull/7500

### CODE REVIEW

**Review by g11tech — APPROVED** [review]
approving the PR as all the comments from the parent EIP PR seemed to be addressed :

https://github.com/ethereum/EIPs/pull/7500 cc @SamWilsn

**Review by g11tech — APPROVED** [review]

File: ERCS/erc-7496.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+Onchain traits can be used by contracts in a variety of different scenarios. For example, a contract that wants to entitle a token to a consumable benefit (e.g. a redeemable) can robustly reflect that onchain. Marketplaces can allow bidding on these tokens based on the trait value without having to rely on offchain state and exposing users to frontrunning attacks. The motivating use case behind this proposal is to protect users from frontrunning attacks on marketplaces where users can list NFTs with certain traits where they are expected to be upheld during fulfillment.
+
+This interface for traits was chosen instead of relying on using regular `getFoo()` and `setFoo()` style functions to allow for brevity in defining, setting, and getting traits. Otherwise, contracts would need to know both the getter and setter function selectors including the parameters that go along with it. In defining general but explicit get and set functions, the function signatures are known and only the trait key and values are needed to query and set the values. Contracts can also add new traits in the future without needing to modify contract code.
```
Comment (SamWilsn) [inline ERCS/erc-7496.md:24]: This paragraph belongs in the rationale section, since it's explaining a technical choice made within this proposal.

Comment (ryanio) [reply ERCS/erc-7496.md]: moved in https://github.com/ethereum/ERCs/pull/26/commits/e1f3d8bb0c343a29c2110e6c4c4b8139cb5fa82b, thanks!

**Comment by ryanio** [comment]
test comment to trigger `Auto Review Bot Trigger / Trigger`

**Comment by SamWilsn** [comment]
Might be stuck because of your assets directory. I _believe_ it's supposed to be `../assets/erc-7496`.

**Review by g11tech — APPROVED** [review]
approving following @SamWilsn 's previous approval prior to merging woes

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**